src/py/flwr/cli/run/run.py

```diff
@@ -113,7 +113,7 @@ def run(
 
 
 def _run_with_superexec(
-    federation: Dict[str, str],
+    federation: Dict[str, Any],
```

Comment:
Shouldn't this type match the proto type? We'll expand to other value types in line with other configs (run config, executor config, node config)

Reply:
@danieljanes This is `Dict[str, Any]` because it is the return type of the `validate_and_load_config` function. The effective type will be determined by the `flatten_dict` function. Currently this means string only, but once, the user config type is merged, it will be `UserConfig`. Would you prefer the conversion to be done before the call to this function?

Reply:
makes sense - let's merge `UserConfig` first then. are you asking about the `flatten_dict` conversion?

Reply:
@danieljanes Yes